storage/rocksdb/CMakeLists.txt

```diff
@@ -27,11 +27,16 @@ IF (NOT CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
   RETURN ()
 ENDIF ()
 
-# Relax unused variable warnings for release modes
-MY_CHECK_CXX_COMPILER_FLAG("-Wunused-variable" HAVE_UNUSED_VARIABLE)
-IF(HAVE_UNUSED_VARIABLE)
-  SET(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -Wno-unused-variable")
-  SET(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -Wno-unused-variable")
+# Suppress warnings for gcc-9
+IF(CMAKE_COMPILER_IS_GNUCXX AND
+   (CMAKE_CXX_COMPILER_VERSION VERSION_EQUAL 9.0 OR CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 9.0))
```

Comment:
```suggestion
   NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.0)
```

Reply:
Good catch, but we already merged VERSION_EQUAL OR VERSION_GREATER in 5.7